Makefile

```diff
@@ -35,7 +35,11 @@ PKG_MANAGER ?= $(shell command -v dnf yum|head -n1)
 # ~/.local/bin is not in PATH on all systems
 PRE_COMMIT = $(shell command -v bin/venv/bin/pre-commit ~/.local/bin/pre-commit pre-commit | head -n1)
 
-SOURCES = $(shell find . -path './.*' -prune -o -name "*.go")
+# This isn't what we actually build; it's a superset, used for target
+# dependencies. Basically: all *.go files, except *_test.go, and except
+# anything in a dot subdirectory. If any of these files is newer than
+# our target (bin/podman{,-remote}), a rebuild is triggered.
+SOURCES = $(shell find . -path './.*' -prune -o \( -name '*.go' -a ! -name '*_test.go' \) -print)
```

Comment:
For what would we need the `-prune` if we just use `-type f` and use only go files which are non tests? I guess the `-print` is obsolete in that case too, right?

Reply:
The `-prune` is what excludes dot-directories; see #5880. You and I may not have dot-directories in our work environment, but the originator of that PR does, and it seems reasonable to me to exclude those.

Reply:
Makes sense.